judoscale/celery/collector.py

```diff
@@ -0,0 +1,79 @@
+import json
+from threading import Thread
+from typing import List, Optional, Set
+
+from celery import Celery
+from kombu import Connection
```

Comment:
Do we need to include this in the project dependencies somewhere?

This actually reminds me that while this PR includes README instructions for *using* the Celery adapter, there's nothing specific to *installing* the Celery adapter. We'd discussed using the "extras" feature for each framework, right? For some reason I thought we'd done that for Django and Flask, but perhaps not.

Reply:
> Do we need to include this in the project dependencies somewhere?

If you're asking about `kombu`, then no, we don't need to include it in the project's dependencies. `kombu` is a dependency of `celery` and will be installed as a transitive dependency automatically.

> there's nothing specific to installing the Celery adapter

This library is currently still installable with the equivalent of `pip install judoscale-python`. Once we switch to the "extras" approach we'll update the README to include instructions for installing the specific integrations.

> We'd discussed using the "extras" feature for each framework, right? For some reason I thought we'd done that for Django and Flask, but perhaps not.

Yep, we have discussed it. But we haven't switched over to using extras yet. We could tackle this next, once the Celery work is merged.

Reply:
Thanks. I couldn't remember where we landed on that. I do think it would be good to tackle that next before we add another integration.